src/openlcb/IfCan.cxx

```diff
@@ -382,8 +382,9 @@ class AMEGlobalQueryHandler : public StateFlowBase,
     {
         while (nextIndex_ < if_can()->local_aliases()->size())
         {
-            if (if_can()->local_aliases()->retrieve(
-                    nextIndex_, nullptr, nullptr))
+            NodeID n;
+            if (if_can()->local_aliases()->retrieve(nextIndex_, &n, nullptr) &&
```

Comment:
Is order of operations within an if statement guaranteed by the language standard? Otherwise, n could be used uninitialized.

Reply:
Yes, an `&&` operator is always evaluated left to right and it has shortcut evaluation by definition of the C standard.